docs/sql_reference/src/asciidoc/_chapters/sql_utilities.adoc

```diff
@@ -650,6 +650,111 @@ Tables must be created/dropped/altered through Hive itself.
 [[load_examples]]
 === Examples of LOAD
 
+* This example demonstrates the LOAD statement continues loading data from source_table into target_table after ignorable data conversion error.

```

Comment:
Grammar is not quite right... adding the word "how" just before "the LOAD statement" will fix it. But this doesn't quite match the content below... the content shows what happens when we don't supply CONTINUE ON ERROR, then it shows a contrasting example with CONTINUE ON ERROR. Possible wordsmith: "The examples below demonstrate how the LOAD statement behaves without and with CONTINUE ON ERROR, when ignorable data conversion errors occur."

Reply:
Thanks Dave, your comments have been incorporated :)